### PR TITLE
migrate dotnet-template-samples repo

### PR DESCRIPTION
**Pull request by YuliiaKovalova**
Related to https://github.com/dotnet/templating/issues/2522#event-7855091718

Migrate https://github.com/dotnet/dotnet-template-samples to this dotnet/templating repo

### CODE REVIEW

File: dotnet-template-samples/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# .NET Template Samples
```
Comment (vlada-shubina) [inline dotnet-template-samples/README.md:1]: this readme has to be adjusted. We can skip it for now, on adjust with suggestions below.

File: dotnet-template-samples/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+This repository contains samples of templates created using the [Template Engine](https://github.com/dotnet/templating/). Custom templates can be
+used from `dotnet new` today. We are working on integrating these templates into Visaul Studio as well.
```
Comment (vlada-shubina) [inline dotnet-template-samples/README.md:4]:
```suggestion
This folder contains samples of templates done for template engine.
These templates are supported by `dotnet new` and Visual Studio. 
For details, on how to configure the templates to be used in Visual Studio, refer to [this article]( https://aka.ms/netcore-templates).
```

File: dotnet-template-samples/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+For more general info on `dotnet new`, and the Template Engine, see the following.
+ - [Announcing .NET Core Tools 1.0](https://blogs.msdn.microsoft.com/dotnet/2017/03/07/announcing-net-core-tools-1-0/)
+ - [How to create your own templates for dotnet new](https://blogs.msdn.microsoft.com/dotnet/2017/04/02/how-to-create-your-own-templates-for-dotnet-new/)
```
Comment (vlada-shubina) [inline dotnet-template-samples/README.md:8]:
```suggestion
For more general info on the authoring, see:
- [How to create custom templates] (https://learn.microsoft.com/en-us/dotnet/core/tools/custom-templates)
- [Template authoring docs in dotnet/templating Wiki](https://github.com/dotnet/templating/wiki)
```

File: dotnet-template-samples/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## Contributing
+
+We would love community contributions here.
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) for information on contributing to this project.
+
+This project has adopted the code of conduct defined by the [Contributor Covenant](http://contributor-covenant.org/) 
+to clarify expected behavior in our community. For more information, see the [.NET Foundation Code of Conduct](http://www.dotnetfoundation.org/code-of-conduct).
```
Comment (vlada-shubina) [inline dotnet-template-samples/README.md:17]:
```suggestion
See [CONTRIBUTING.md](../CONTRIBUTING.md) for information on contributing to this project.
```

File: dotnet-template-samples/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+This repository contains samples of templates created using the [Template Engine](https://github.com/dotnet/templating/). Custom templates can be
+used from `dotnet new` today. We are working on integrating these templates into Visaul Studio as well.
+
+For more general info on `dotnet new`, and the Template Engine, see the following.
+ - [Announcing .NET Core Tools 1.0](https://blogs.msdn.microsoft.com/dotnet/2017/03/07/announcing-net-core-tools-1-0/)
+ - [How to create your own templates for dotnet new](https://blogs.msdn.microsoft.com/dotnet/2017/04/02/how-to-create-your-own-templates-for-dotnet-new/)
+
+## Contributing
+
+We would love community contributions here.
```
Comment (vlada-shubina) [inline dotnet-template-samples/README.md:12]:
```suggestion
We would love community contributions here. Please feel free to open the PRs to add more examples or create an [issue](https://github.com/dotnet/templating/issues/new) requesting a new example.
```

File: dotnet-template-samples/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+This project is licensed with the [MIT license](LICENSE).
+
+## Related Projects
+
+You should take a look at these related projects:
+
+- [Template Engine](https://github.com/dotnet/templating/)
+- [.NET Core](https://github.com/dotnet/core)
+- [ASP.NET](https://github.com/aspnet)
+- [Mono](https://github.com/mono)
```
Comment (vlada-shubina) [inline dotnet-template-samples/README.md:30]:
```suggestion
## Template Content Repositories

[.NET default templates](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates) are located in several repositories. You can use them as examples of more complicated templates.

The templates are located in the following repositories:

| Templates | Repository |
|---|---|
|Common project and item templates|[dotnet/sdk](https://github.com/dotnet/sdk)|
|ASP.NET and Blazor templates|[dotnet/aspnetcore](https://github.com/dotnet/aspnetcore)|
|ASP.NET Single Page Application templates| [dotnet/spa-templates](https://github.com/dotnet/spa-templates)|
|WPF templates|[dotnet/wpf](https://github.com/dotnet/wpf)|
|Windows Forms templates|[dotnet/winforms](https://github.com/dotnet/winforms)|
|Test templates|[dotnet/test-templates](https://github.com/dotnet/test-templates)|
|MAUI templates|[dotnet/maui](https://github.com/dotnet/maui)|

```

**Review by vlada-shubina — CHANGES_REQUESTED** [review]
Readme.md should be adjusted or removed - it's fairly outdated.

Further on (after this PR), please create the PRs from your fork. We are developing in our own forks and making the PRs from them.

File: dotnet-template-samples/README.md (hunk 1 above)
Comment (YuliiaKovalova) [reply dotnet-template-samples/README.md:1]: done

**Review by vlada-shubina — APPROVED** [review]

**Pull request merged**